packages/core/src/external/athenahealth/index.ts

```diff
@@ -154,8 +165,10 @@ class AthenaHealthApi {
           password: this.config.clientSecret,
         },
       });
-
-      this.twoLeggedAuthToken = response.data.access_token;
+      if (!response.data) throw new MetriportError("No body returned from token endpoint");
+      const tokenData = athenaClientJwtTokenResponseSchema.parse(response.data);
+      this.twoLeggedAuthToken = tokenData.access_token;
+      this.twoLeggedAuthTokenExp = new Date(Date.now() + +tokenData.expires_in * 1000).getTime();
```

Comment:
doble plus/addition

Reply:
Will wrap it so it's `(+tokenData.expires_in)`

Reply:
Done.